src/main/java/io/cdap/plugin/dlp/SensitiveRecordFilter.java

```diff
@@ -261,6 +293,30 @@ private DlpServiceSettings getSettings() throws IOException {
     @Description("Error handling of record")
     private String onError;
 
+    @Macro
+    @Name("dlp-override-enabled")
+    @Description("Use custom DLP service endpoint")
+    private Boolean dlpOverrideEnabled;
+
+    @Macro
+    @Name(DLP_HOST)
+    @Nullable
+    @Description("DLP host")
+    private String dlpHost;
+
+    @Macro
+    @Name(DLP_PORT)
+    @Nullable
+    @Description("DLP port")
+    private String dlpPort;
+
+    @Macro
+    @Name(DLP_TLS_ENABLED)
+    @Nullable
+    @Description("When enabled, credentials are sent to DLP over TLS. "
+        + "When disabled, credentials are not sent and TLS is not established.")
```

Comment:
We should update this to be similar to the docs in the other file

Reply:
Ok